scenarios/azure/aks/run.go

```diff
@@ -13,11 +23,98 @@ func Run(ctx *pulumi.Context) error {
 		return err
 	}
 
-	_, kubeConfig, err := aks.NewCluster(env, "aks", nil)
+	cluster, kubeConfig, err := aks.NewCluster(env, "aks", nil)
 	if err != nil {
 		return err
 	}
 
+	// Export cluster’s properties
 	ctx.Export("kubeconfig", kubeConfig)
```

Comment:
I'd suggest to pu these modifications after https://github.com/DataDog/test-infra-definitions/pull/484 to directly implement the Kubernetes component for AKS.

Reply:
Do you have an estimation of when it should be merged ?